Bottom: f5c3d6f4c537e73e679e11bb747cf25492430357
Top:    059584ef7ec1a14ac709dd0550fc240d4d988def
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 20:27:46 +0000

hid/lesstif: Use an intervening size_t casts to aovid compiler warnings

This is to avoid the compiler warning us what we already know.. that we
are passing an integer value cast into what might be a different sized
pointer. (On 64 bit builds).


---

diff --git a/src/hid/lesstif/dialogs.c b/src/hid/lesstif/dialogs.c
index b3cc375..6c7a2de 100644
--- a/src/hid/lesstif/dialogs.c
+++ b/src/hid/lesstif/dialogs.c
@@ -1791,7 +1791,7 @@ lesstif_attributes_need_rows (int new_max)
       XtManageChild (attr_row[attr_max_rows].del);
       XtAddCallback (attr_row[attr_max_rows].del, XmNactivateCallback,
 		     (XtCallbackProc) attributes_delete_callback,
-		     (XtPointer) attr_max_rows);
+		     (XtPointer) (size_t) attr_max_rows);
 
       n = 0;
       stdarg (XmNresizeWidth, true);
@@ -1862,7 +1862,7 @@ attributes_delete_callback (Widget w, void *v, void *cbs)
   int i, n;
   Widget wn, wv;
 
-  n = (int) v;
+  n = (int) (size_t) v;
 
   wn = attr_row[n].w_name;
   wv = attr_row[n].w_value;
